set.mm

```diff
@@ -51013,8 +51013,12 @@ We have not yet defined relations ( ~ df-rel ), but here we introduce a few
        ` F = { <. 2 , 6 >. , <. 3 , 9 >. } -> ran F = { 6 , 9 } ` ( ~ ex-rn ).
        Contrast with domain (defined in ~ df-dm ).  For alternate definitions,
        see ~ dfrn2 , ~ dfrn3 , and ~ dfrn4 .  The notation " ` ran ` " is used
-       by Enderton; other authors sometimes use script R or script W.
-       (Contributed by NM, 1-Aug-1994.) $)
+       by Enderton; other authors sometimes use script R or script W. The range
```

Comment:
```suggestion
       by Enderton. The range
```
I think this is superfluous (and not before long, we would have to add the whole alphabet).

Reply:
> I think this is superfluous (and not before long, we would have to add the whole alphabet).

Maybe "W" comes from the German "Wertebereich". But you are right, thee is not much value in this hint, so I will remove it.

Reply:
Done.